SAMtags.tex

```diff
@@ -246,25 +250,26 @@ \subsection{Barcodes}
 The {\tt BC} tag should match the {\tt QT} tag in length. 
 In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and places a hyphen (`{\tt -}') between the barcodes from the same template. 
 
-\item[QT:Z:\tagvalue{qualities}] 
+\item[QT:Z:\tagvalue{qualities}]
 Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. 
 Same encoding as {\sf QUAL}, i.e., Phred score + 33.
-In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with spaces (`{\tt \textvisiblespace}') between the different strings from the same template. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the quality strings with spaces (`{\tt \textvisiblespace}') between the different strings from the same template.
 
-\item[RX:Z:\tagvalue{sequence+}]
-Sequence bases from the unique molecular identifier. 
-These could be either corrected or uncorrected. Unlike {\tt MI}, the value may be non-unique in the file. 
-Should be comprised of a sequence of bases. 
-In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes with a hyphen (`{\tt -}') between the different barcodes.
+\item[CB:Z:\tagvalue{str}]
+Cell identifier based on the cellular-barcode sequence ({\tt CR}) that may be optionally i) error corrected and ii) followed by a dash (`{\tt -}') and one or more alphanumeric characters to form an identifier. 
+In the case of the cellular barcode ({\tt CR}) being based on multiple barcode sequences the recommended implementation concatenates all the (corrected or uncorrected) barcodes with a hyphen (`{\tt -}') between the different barcodes. 
+Notwithstanding errors (e.g., sequencing), all reads from a single cell are expected to have the same {\tt CB} tag.
 
-If the bases represent corrected bases, the original sequence can be stored in {\tt OX} (similar to {\tt OQ} storing the original qualities of bases.)
+\item[CR:Z:\tagvalue{sequence+}]
+Cellular barcode. Sequence bases of the cellular-barcode as reported by the sequencer (uncorrected) with the respective quality scores (optionally) stored in {\tt CY}. 
```

Comment:
Brackets are probably unnecessary.  Eg "Uncorrected sequence bases... quality scores optionally stored..".  However it's tiny nit-picking and I'm OK with it either way.